src/pybind/mgr/nfs/module.py

```diff
@@ -34,6 +34,7 @@ def _cmd_nfs_export_create_cephfs(
             readonly: Optional[bool] = False,
             client_addr: Optional[List[str]] = None,
             squash: str = 'none',
+            sectype: Optional[List[str]] = None,
```

Comment:
Document it here?
https://github.com/ceph/ceph/blob/main/doc/mgr/nfs.rst#create-cephfs-export

and also maybe here,
https://github.com/ceph/ceph/blob/main/doc/mgr/nfs.rst#create-or-update-export-via-json-specification

Reply:
OK, I was a little iffy on documenting it if the feature is only useful on (future) rook deployments but we can certainly add it with a caveat.

Reply:
Docs added.